man-roxygen/arg_fdatask_pars.R

```diff
@@ -0,0 +1,12 @@
+#' @param fd.features [\code{list}]\cr Optional. Named list of column indices or
+#'   column names of the passed data. Each list entry defines one functional covariate through
+#'   either a character vector of column names or an integer vector of column
+#'   indices. The list names specify the names of the functional covariates. All
+#'   columns that are not referenced in \code{fd.features} are scalar variables.
+#'   Default is \dQuote{fd1} which means we assume that all columns form one
+#'   functional covariate.
+#' @param fd.grids [\code{list}]\cr Optional. Named list of grids over which the
```

Comment:
are these used always together? Otherwise maybe create two templates, also makes it clearer to read in the respective files